### ISSUE TITLE
using deprecated config access

### ISSUE THREAD

**Issue by chrisbitmead** [nerdErg/grails-shiro#19]
on ShiroGrailsPlugin.groovy line 199 it accesses configuration using the deprecated direct access

```
grailsApplication.config.info.app.name
```
instead of:
```
grailsApplication.config.getProperty('info.app.name')
```

I think it was very dumb for grails to deprecate this, but it is what it is.

**Comment by pmcneil** [comment]
This method has some checks for existence and the ability to provide a default I believe, so it's a good thing. Reminds me I need to update this plugin and release it !

**Comment by chrisbitmead** [comment]
The groovy language also has syntax to check for existence and provide a default.

**Comment by sbglasius** [comment]
@chrisbitmead In Grails 7 it was un-deprecated, because we felt it was dumb too.

**Comment by miq** [comment]
@sbglasius So we just leave it that way and close this?

**Comment by xpusostomos** [comment]
Yes, I saw that, although I'm yet to go to grails 7 myself, so close away, and I'm glad someone saw sense.
